src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
```

Comment:
should be singular `removeRecipeToCook` since it removes one recipe, not multiple